gst/nnstreamer/tensor_transform/tensor_transform.c

```diff
@@ -689,20 +689,34 @@ gst_tensor_transform_set_option_data (GstTensorTransform * filter)
     }
     case GTT_STAND:
     {
-      gchar **strv = NULL;
-
-      strv = g_strsplit (filter->option, ":", -1);
+      gchar **str_options = NULL;
+      gchar **options_strv = NULL;
+      guint str_options_num;
 
-      filter->data_stand.mode = gst_tensor_transform_get_stand_mode (strv[0]);
       filter->data_stand.out_type = _NNS_END;
-      if (g_strv_length (strv) > 1)
-        filter->data_stand.out_type = gst_tensor_get_type (strv[1]);
+      filter->data_stand.per_channel = FALSE;
 
-      g_strfreev (strv);
+      str_options = g_strsplit (filter->option, ",", -1);
+      str_options_num = g_strv_length (str_options);
+
+      options_strv = g_strsplit (str_options[0], ":", -1);
+      filter->data_stand.mode =
+          gst_tensor_transform_get_stand_mode (options_strv[0]);
+      filter->data_stand.out_type = gst_tensor_get_type (options_strv[1]);
+
+      if (str_options_num > 1) {
+        options_strv = g_strsplit (str_options[1], ":", -1);
+        if (g_str_equal (options_strv[1], "true")) {
+          filter->data_stand.per_channel = TRUE;
+        }
+      }
+
+      g_strfreev (options_strv);
```

Comment:
prevent mem leak, move this to line 706 and add add g_strfreev (options_strv); in 712.

Reply:
Or how about adding loop to avoid duplicated code
```
for (i = 0; i < str_options_num; i++) {
  options_strv = g_strsplit (str_options[i], ":", -1);

  if (g_ascii_strcasecmp (options_strv[0], "default") == 0 || 
      g_ascii_strcasecmp (options_strv[0], "dc-average") == 0) {
    filter->data_stand.mode = gst_tensor_transform_get_stand_mode (options_strv[0]);
    filter->data_stand.out_type = gst_tensor_get_type (options_strv[1]);
  } else if (g_ascii_strcasecmp (options_strv[0], "per-channel") == 0) {
    if (g_ascii_strcasecmp (options_strv[1], "true") == 0)
      filter->data_stand.per_channel = TRUE;
  } else {
    ml_logw ("unknown %s", options_strv[0]);
  }

  g_strfreev (options_strv);
}
```

Reply:
Cool, Looks neat. I updated the commit after this.